src/native/corehost/corehost.proj

```diff
@@ -78,7 +78,7 @@
       <BuildArgs Condition="'$(KeepNativeSymbols)' != 'false'">$(BuildArgs) -keepnativesymbols</BuildArgs>
       <BuildArgs Condition="'$(CrossBuild)' == 'true'">$(BuildArgs) -cross</BuildArgs>
       <BuildArgs Condition="'$(Compiler)' != ''">$(BuildArgs) $(Compiler)</BuildArgs>
-      <BuildArgs Condition="'$(CMakeArgs)' != ''">$(BuildArgs) $(CMakeArgs)</BuildArgs>
+      <BuildArgs Condition="'$(CMakeArgs)' != ''">$(BuildArgs) -cmakeargs "$(CMakeArgs)"</BuildArgs>
```

Comment:
shouldn't this use `&quot;` instead of literal quotes?

Reply:
> shouldn't this use &quot; instead of literal quotes?

This is content, not the attribute value.

> Yes. This file has some examples: lines 50, 92, 93.

See line 74 and 75.